prow/jobs/kyma-project/kyma/tests/components/application-connector/central-application-connectivity-validator/tests-validator.yaml

```diff
@@ -0,0 +1,113 @@
+          repo: local-kyma
+          path_alias: github.com/kyma-incubator/local-kyma
+          base_ref: main
+        - org: kyma-project
+          repo: test-infra
+          path_alias: github.com/kyma-project/test-infra
+          base_ref: main
+      spec:
+        containers:
+          - image: "europe-docker.pkg.dev/kyma-project/prod/testimages/e2e-dind-k3d:v20230713-3ebd2748"
+            securityContext:
+              privileged: true
+              seccompProfile:
+                type: Unconfined
+              allowPrivilegeEscalation: true
+            command:
+              - "/init.sh"
+            args:
+              - "bash"
+              - "-c"
+              - "make -C tests/components/application-connector/hack/ci k3d-validator-tests"
+            resources:
+              requests:
+                memory: 1Gi
+                cpu: 500m
+              limits:
+                cpu: 1
+                memory: 4Gi
+
+postsubmits:
+  kyma-project/kyma:
+    - name: post-main-kyma-integration-k3d-app-conn-validator
+      annotations:
+        description: "Kyma integration job on k3d for testing application connectivity validator."
+        owner: "framefrog"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "post-main-kyma-integration-k3d-app-conn-validator"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-build-main: "true"
+        preset-gc-project-env: "true"
+        preset-kyma-guard-bot-github-token: "true"
+        preset-kyma-integration-app-connector-tests-app-validator: "true"
+        preset-kyma-integration-central-app-connectivity-enabled: "true"
+        preset-kind-volume-mounts: "true"
+        preset-dind-enabled: "true"
+      skip_report: false
+      decorate: true
+      path_alias: github.com/kyma-project/kyma
+      cluster: trusted-workload
+      max_concurrency: 10
+      branches:
+        - ^master$
+        - ^main$
+      extra_refs:
+        - org: kyma-incubator
+          repo: local-kyma
+          path_alias: github.com/kyma-incubator/local-kyma
+          base_ref: main
+        - org: kyma-project
+          repo: test-infra
+          path_alias: github.com/kyma-project/test-infra
+          base_ref: main
+      spec:
+        containers:
+          - image: "europe-docker.pkg.dev/kyma-project/prod/testimages/e2e-dind-k3d:v20230713-3ebd2748"
+            securityContext:
+              privileged: true
+              seccompProfile:
+                type: Unconfined
+              allowPrivilegeEscalation: true
+            command:
+              - "/init.sh"
+            args:
+              - "bash"
+              - "-c"
+              - "make -C tests/components/application-connector/hack/ci k3d-validator-tests"
+            resources:
+              requests:
+                memory: 1Gi
+                cpu: 500m
+              limits:
+                cpu: 1
+                memory: 4Gi
```

Comment:
When you run a presubmit job, Prow automatically fetches latest revision of trunk branch (in this case main) and rebases your changes introduced in a PR. This way you will always test the revision you are going to have after merge. I see no sense in running postsubmit which is doing exactly the same thing as presubmit.